test/cases/wasm/finalize-exports-issue-8261/node_modules/side-effect-free/module.wat

```diff
@@ -0,0 +1,5 @@
+(module
+  (func (export "a") (result i32)
+    (i32.const 1)
+  )
+)
```

Comment:
This would also work for the test:

```wat
(module
  (func (export "a"))
)
```

Reply:
Thanks, sent a new PR #8286 to improve it.